ecogvis/functions/misc_dialogs.py

```diff
@@ -175,8 +192,10 @@ def __init__(self, parent, type):
         self.exec_()
 
     def run_estimation(self):
-        self.text.setText("Calculating the Power Spectral Density for " + self.type + " data.\n"
-                          "Please wait, this might take around 5 minutes.")
+        self.text.setText(
+            "Calculating the Power Spectral Density for " + self.type + " "
+                                                                        "data.\n" +
+            "Please wait, this might take around 5 minutes.")
```

Comment:
again, why it is disposed like that?

Reply:
yeah, this looks like a PyCharm type of automatic reformatting to fit PEP8 style with 79 chars per line. they may look strange but it shouldn't have resulted in any errors right?

Reply:
that's right, it shouldn't incur any errors